modules/tensor_mechanics/src/kernels/InertialTorque.C

```diff
@@ -14,7 +14,7 @@ InputParameters
 validParams<InertialTorque>()
 {
   InputParameters params = validParams<TimeKernel>();
-  params.addClassDescription("Kernel for interial torque: density * displacement x acceleration");
+  params.addClassDescription("Kernel for interial torque: density * displacement * acceleration");
```

Comment:
No, leave the "x" there: it indicates a cross product instead of a multiplication